src/components/Character/Character.jsx

```diff
@@ -55,29 +67,37 @@ export default function Character(props){
 			aria-roledescription="slide"
 			aria-label={`Character ${index + 1} of ${characterData.length}.`}
 			aria-hidden={ hidden }>
-			<article>
-				<h1 id={`character__${id}`}>
-					{ name }
-				</h1>
-				<h2>
-					{ nickname }
-				</h2>
-				<h3>
-					{ actor }
-				</h3>
-				<p>
-					{ body }
-				</p>
-				<img 
-					src={ src } 
-					alt={ description }
-					style={ mask }
-				/>
-
-				{ prevCharName && RENDER.button(prevCharName, -1) }
-				{ nextCharName && RENDER.button(nextCharName, +1) }
-
+			<article className={ s.container }>
+				<div className={ s.content }>
+					<div className={ s.headings }>
+						<h1 
+							className={ s.name } 
+							id={`character__${id}`}
+						>
+							{ name }
+						</h1>
+						<h2 className={ s.nickname } >
+							{ nickname }
+						</h2>
+						<h3 className={ s.actor } >
+							{ actor }
+						</h3>
+					</div>
+					<div className={ s.body } > 
+						{ body }
+					</div>	
+				</div>
+				<nav className={ s.controls }>
+					{ prevButton }
+					{ nextButton }
+				</nav>	
 			</article>
+			<HeroImage 
+				heroId={ heroId }
```

Comment:
```suggestion
				id={ heroId }
```
Feel like I might have mentioned this in another one - apologies for missing it if not - we already have 'hero' in the component name so don't really need to say `heroId` as the name of the prop

Reply:
You did mention it before but I thought using id in HeroImage would cause issues. But it appears that id is just treated like any other prop?